src/assets/css/bases/typography.css

```diff
@@ -1,18 +1,36 @@
 /*-----------------------------------------*\
   #TYPOGRAPHY ELEMENT
 \*-----------------------------------------*/
-
+p {
```

Comment:
Why do you style for p tag? Keep in your mind, never style directly on the HTML element. Because we can't make sure it's not changed in the future.

Reply:
Because I see in Figma the texts in the p tag are all font-size: 16px, font-weight: 
 400 so I grouped them for convenience.